Rakefile

```diff
@@ -68,6 +68,17 @@ task :release, [:version] => [:prepare_release] do |t, args|
   mkdir_p path
   release(package, args.version)
 
+  # Push on master the new Dockerfile
+  d = Git.open("#{ENV['GOPATH']}/src/github.com/ovh/svfs", :log => Logger.new(STDOUT))
```

Comment:
Generate the new docker file in the release commit, not with a new commit on master.